src/components/Footer/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './styles.scss';
+
+export default function Footer() {
+  return (
+    <section className='footer-container'>
+      <div>Footer</div>
+    </section>
```

Comment:
`<section>` element should have heading(`h1-h6`). Since there's no heading, could we avoid unnecessary nested `<div>`? What do you think?

Source: https://stackoverflow.com/questions/6939864/what-is-the-difference-between-section-and-div
```suggestion
    <div className='footer-container'>Footer</div>
```

Reply:
Okay thanks 🤓